tests/system/small/test_dataframe.py

```diff
@@ -1161,6 +1161,23 @@ def test_apply_series_scalar_callable(
     pandas.testing.assert_series_equal(bf_result, pd_result)
 
 
+def test_apply_from_read_gbq_function(dataset_id_permanent):
+    @bpd.remote_function()
```

Comment:
We don't need to deploy a new remote function (it is costly, takes around 1 minute). We can write the test using existing functions screen/8yHKQ5KibqDMLvL

Reply:
Makes sense. Used an existing function instead.